code/game/objects/items/implants/implant_misc.dm

```diff
@@ -83,3 +83,62 @@
 		if (!healthstring)
 			healthstring = "ERROR"
 		return healthstring
+
+// Radio implant
+
+/obj/item/implant/radio
+	name = "internal radio implant"
+	activated = TRUE
+	var/obj/item/device/radio/radio
+	var/radio_key
+	var/subspace_transmission = FALSE
+	icon = 'icons/obj/radio.dmi'
+	icon_state = "walkietalkie"
+
+/obj/item/implant/radio/activate()
+	. = ..()
+	// needs to be GLOB.deep_inventory_state otherwise it won't open
+	radio.ui_interact(usr, "main", null, FALSE, null, GLOB.deep_inventory_state)
+
+/obj/item/implant/radio/Initialize(mapload)
+	. = ..()
+
+	radio = new(src)
+	// almost like an internal headset, but without the
+	// "must be in ears to hear" restriction.
+	radio.name = "internal radio"
+	radio.subspace_transmission = subspace_transmission
+	radio.canhear_range = 0
+	if(radio_key)
+		radio.keyslot = new radio_key
+	radio.recalculateChannels()
+
+/obj/item/implant/radio/mining
+	radio_key = /obj/item/device/encryptionkey/headset_cargo
+
+/obj/item/implant/radio/syndicate
+	desc = "Are you there God? It's me, Syndicate Comms Agent."
+	radio_key = /obj/item/device/encryptionkey/syndicate
+	subspace_transmission = TRUE
+
+/obj/item/implant/radio/slime
```

Comment:
Probably pasted here by mistake.

Reply:
Actually, nevermind. Might as well leave it as an admin thing until this is used. Ignore that.